Description: Disable "vendored" feature
 This feature relies on an identically named feature in libusb1-sys, which has
 also been disabled as it requires vendored library C code, which we want to
 exclude from Debian packages.
Author: Arnaud Ferraris <aferraris@debian.org>
Forwarded: not-needed
Last-Update: 2023-11-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,8 +47,5 @@
 [dev-dependencies.usb-ids]
 version = "1.2023.0"
 
-[features]
-vendored = ["libusb1-sys/vendored"]
-
 [badges.travis-ci]
 repository = "a1ien/rusb"
